storybook/stories/Examples/ResponsiveContainer/MultiChartFlexbox.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {
+  ResponsiveContainer,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Area,
+  AreaChart,
+  LineChart,
+  Line,
+} from '../../../../src';
+import { pageData } from '../../data';
+import './style.css';
+
+export default {
+  component: AreaChart,
+};
+
+export const MultiChartFlexbox = {
+  render: (_args: Record<string, any>) => {
+    return (
+      <>
+        <p>Resize the window to test ResponsiveContainer</p>
+        <div className="flex-parent">
+          <ResponsiveContainer width="100%" className="flex-child">
+            <AreaChart
+              data={pageData}
+              margin={{
+                top: 10,
+                right: 30,
+                left: 0,
+                bottom: 0,
+              }}
+            >
+              <CartesianGrid strokeDasharray="3 3" />
+              <XAxis dataKey="name" />
+              <YAxis />
+              <Tooltip />
+              <Area type="monotone" dataKey="uv" stroke="#8884d8" fill="#8884d8" />
+            </AreaChart>
+          </ResponsiveContainer>
+          <ResponsiveContainer className="flex-child">
```

Comment:
Trying to add a demo to address [this original issue](https://github.com/recharts/recharts/issues/172) (oldest open issue in repo) - [see repro](https://codesandbox.io/s/area-chart-in-responsive-repro-2023-u48m66).

If everything is in one container you only get the width and height of of the entire container rather than each chart

Reply:
`ResponsiveContainer` can also only have one child